src/ps1/ps1_memory_card.c

```diff
@@ -17,13 +20,41 @@ static volatile int reset;
 static int ignore;
 static uint8_t flag;
 
+static size_t game_id_length;
+static char received_game_id[0x10];
+static volatile uint8_t mc_pro_command;
+
+
 typedef struct {
     uint32_t offset;
     uint32_t sm;
 } pio_t;
 
 static pio_t cmd_reader, dat_writer;
 static volatile int mc_exit_request, mc_exit_response, mc_enter_request, mc_enter_response;
+static void __time_critical_func(clean_title_id)(const uint8_t* const in_title_id, char* const out_title_id, const size_t in_title_id_length);
```

Comment:
no need to forward declare this function since it's not being used prior to being defined